collections/vec.go

```diff
@@ -0,0 +1,141 @@
+package collections
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"cosmossdk.io/collections/codec"
+)
+
+var (
+	// ErrEmptyVec is returned when trying to pop an element from an empty Vec.
+	ErrEmptyVec = errors.New("vec is empty")
+	// ErrOutOfBounds is return when trying to do an operation on an index that is out of bounds.
```

Comment:
```suggestion
	// ErrOutOfBounds is returned when trying to do an operation on an index that is out of bounds.
```
? I think this is a mistake but not 100% sure

Reply:
thanks